shared-bindings/busio/__init__.c

```diff
@@ -87,10 +87,13 @@
 
 STATIC const mp_rom_map_elem_t busio_module_globals_table[] = {
     { MP_ROM_QSTR(MP_QSTR___name__), MP_ROM_QSTR(MP_QSTR_busio) },
-    { MP_ROM_QSTR(MP_QSTR_I2C),   MP_ROM_PTR(&busio_i2c_type) },
     { MP_ROM_QSTR(MP_QSTR_SPI),   MP_ROM_PTR(&busio_spi_type) },
+// a pIRKey does not need these. Remove them to make room for frozen modules.
+#ifndef PIRKEY_M0
```

Comment:
Lets not remove these. I don't want modules besides `board` to have different members. Either add them back or have the constructors all raise an error with a friendly error message (versus NameError).

Reply:
I worked on this, trying a couple of different ways. It was a maze of `#ifdef'`s because I have to remove a bunch of static routines in `shared-bindings/{I2C.c,UART.c}`: otherwise I get "defined but not used" errors. And, ultimately, you can't even call the stub constructors (which would raise `NotImplementedError`) anyway, because there are no reasonable pins available to use. So I'm kind of inclined to leave it the way it was.

Reply:
btw, waiting for Limor to test actual functionality

Reply:
You could add a common-hal method that the shared-bindings code calls. You have room to leave these in right?

Reply:
There's not enough flash if I leave I2C and UART in. That's the main reason I removed them -- it wasn't because there are no pins for the functionality. To gain space, I have to empty the locals dict, etc. so that the routines aren't compiled in, and that involved a lot of #ifdef'ing.

I don't think anyone is going to try I2C or UART on the pirkey because there's nothing to connect to to use them. We need to keep SPI for the on-board DotStar.

Reply:
They are pretty big, like 6k.